src/lib/sql/sql_translator.hpp

```diff
@@ -51,27 +51,37 @@ class SQLTranslator final {
   static std::shared_ptr<AbstractExpression> translate_hsql_expr(const hsql::Expr& hsql_expr, const UseMvcc use_mvcc);
 
  private:
+  // An expression and its identifiers. This is partly redundant to the SQLIdentifierResolver, but allows similar
+  // expressions with different identifiers (e.g., SELECT COUNT(*) AS cnt1, COUNT(*) AS cnt2 FROM ...).
```

Comment:
Can you describe how these "similar" expressions are similar?

Reply:
Not addressed yet. If you say something is similar, you should explain how it is not equal.

Reply:
Now I (hopefully) understood you. I actually meant "equal" instead of "similar". Good catch.

Reply:
Yes. One more thing that I haven't seen before (and have changed myself): An SQL statement is the entire SELECT thing. In this case, we are talking about equal expressions within the select list of the statement.